docs/bigquery-usage.rst

```diff
@@ -48,17 +48,58 @@ Authorization / Configuration
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_p12('/path/to/creds.p12', 'jrandom@example.com')
 
-- Override the project inferred from the environment by passing an explicit
-  ``project`` to the constructor, or to either of the alternative classmethods:
+
+Projects
+--------
+
+A project is the top-level container in the ``BigQuery`` API:  it is tied
+closely to billing, and can provide default access control across all its
+datasets.  If no ``project`` is passed to the client container, the library
+attempts to infer a project using the environment (including explicit
+environment variables, GAE, and GCE).
+
+To override the project inferred from the environment, pass an explicit
+``project`` to the constructor, or to either of the alternative classmethods:
```

Comment:
You never show the class methods. Also "classmethod" is not a word, consider making it fixed width (`<code>`) font?